### DISCUSSION TITLE
Geschlecht sollte kein Pflichtfeld sein

### DISCUSSION THREAD

**Discussion by tolot27** [openjverein/jverein#336] [General]
Warum ist die Angabe des Geschlechts Pflicht? Mir fällt kein Grund dafür ein, außer zu statistischen Zwecken. Wo kann man das abschalten?

**Comment by tolot27** [comment]
Ah, ich hab zumindest einen Grund gefunden: wenn man eine geschlechtsspezifische Anrede machen möchte. Falls ich nochmal was importieren sollte, werde ich "Ohne Angabe" als Standard setzen, falls ich die Information nicht habe.

**Comment by lenilsas** [comment]
Mich hat das auch schon öfters gestört, dass das Geschlecht ein Pflichtfeld ist wenn ich zB. nur eine Adresse anlegen möchte. Mein Vorschlag wäre, das wir es so ändern, dass "Ohne Angabe" die Vorauswahl ist.

**Reply by tolot27** [reply]
ACK

**Reply by lenilsas** [reply]
in #344 umgesetzt
